collection-scripts/gather_etcd

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+RUNNING_ETCD_POD=$(oc get pod -n openshift-etcd --no-headers|grep Running| grep -o -m 1 '\S*etcd\S*')
+
+function ocp4etcdctl {
+    oc -n openshift-etcd exec -c etcdctl ${RUNNING_ETCD_POD} -- etcdctl $@
+}
+
+if [[ -z $RUNNING_ETCD_POD ]];then
+    echo "ERROR: No running etcd pods found" >&2
+    exit 1
+fi
+
+mkdir -p ${ETCD_LOG_PATH}
+
+echo "INFO: Starting getting etcd information"
+
+# member list
+echo "INFO: Getting etcdctl member list"
+ocp4etcdctl member list -w table > ${ETCD_LOG_PATH}/member_list.txt
+ocp4etcdctl member list -w json > ${ETCD_LOG_PATH}/member_list.json
+
+# endpoint status
+echo "INFO: Getting etcdctl endpoint status"
+ocp4etcdctl endpoint status -w table > ${ETCD_LOG_PATH}/endpoint_status.txt
+ocp4etcdctl endpoint status -w json > ${ETCD_LOG_PATH}/endpoint_status.json
+
+# endpoint health
+echo "INFO: Getting etcdctl endpoint health"
+ocp4etcdctl endpoint health -w table > ${ETCD_LOG_PATH}/endpoint_health.txt
+ocp4etcdctl endpoint health -w json > ${ETCD_LOG_PATH}/endpoint_health.json
+
+# alarm list
+echo "INFO: Getting etcdctl alarm list"
+ocp4etcdctl alarm list > ${ETCD_LOG_PATH}/alarm_list.txt
+ocp4etcdctl alarm list -w json > ${ETCD_LOG_PATH}/alarm_list.json
+
+# List all etcd keys
```

Comment:
This seems like it might be a security issue, in some 'rare' circumstances. 
Should we have a toggle for this? It also seems like a `costly` operation to preform. How much data is this going to pull back? @smarterclayton what is the averaged etcd count size shown from telemetry?

Reply:
My original intention with getting all the keys was to contrast whether users may claim something has been lost and/or some leftover from an unsupported version was present (due to a bug that prevented some api version migration, for example).

However, I did not thought about the security implications, so maybe I can just remove the line